pyproject.toml

```diff
@@ -1,3 +1,36 @@
+[tool.ruff]
+select = [
+    "E",     # pycodestyle errors
+    "W",     # pycodestyle warnings
+    "F",     # pyflakes
+    "C",     # flake8-comprehensions
+    "B",     # flake8-bugbear
+    "ASYNC", # flake8-async
+    "C4",    # flake8-comprehensions
+    "G",     # flake8-logging-format
+    "PL",     # pylint
+]
+ignore = [
```

Comment:
Why such a long list of disabled checks?

Reply:
https://github.com/criteo/sonic-salt-deployer/pull/14

the remaining is code style.